storybook/storybook-react/config/preview.tsx

```diff
@@ -1,4 +1,5 @@
 import '@amsterdam/design-system-tokens/dist/root.css'
+import '@amsterdam/design-system-tokens/dist/compact.css'
```

Comment:
Cascading compact styles over the default theme seems like a good idea because we mainly need new rules for logical properties and font-size. But I fear it will amount in a lot of overrides over time because we redeclare that stuff continuously.

Reply:
> because we redeclare that stuff continuously.

Not really, I think? The definitions for text levels are kind of stable now. I expect that we only need `typography.compact.tokens.json` and `grid.compact.tokens.json` override files for the entire compact theme.